.travis.yml

```diff
@@ -5,8 +5,24 @@ env:
   global:
     - secure: "vM9x0k0SYIgWvrRHOjBZmM+VWp3zGJmUt8dIXEfyJ1gFzWFKuXCP2RvSCtgVc4uNGptm3WocX2MJpPwmI8mA7VrvarmJJf7tuFgdE+UwvuI2Ist1N16LOrEg/ZINncuH9P/is8KVSBXDQm6HsyGMfvOtloaRhxRpT1/+1jT+d0Qfs4EV9lUGUwXD4A/cFysZ5/UpKA6LqgQ4H9onS43tRYuef465m7SnP9qQ0suOnE32767ZM8sTA1cQ3VTcF0m7YujtXLSCsqkhnpaPY4H54iX0IysAeT1LhC6vQ6SE1wl+aS6WVKSTyGBZ1550wh1Ib5wwGNr8szp6OqHbrMvC9E00KR8TuuIPHAW7oRQDV+9tQvb3vq2Q6AYgGzzZL29fCQRbb5Fu6kvEuiRhZdaCXRgwFUDO671hguA3lPZ9Nn81nlijV2UTOf5wfwgmyJ+U2hG57OnnT1X3sz5IXMb6lBM3IF6h1/5Bbkw8pGJBVi49QWit2HdAfj0tPx2Kzs4aMCS4srSFRQKXirlafztPj5eDXBXGiDyzcxsYo3YYreCJXmECFFDWvUqNsMHq91cz09BOMj73UXYt2U9ZMR+x+Jr4dQkBEJaktL7j+3DVd4r31WdKnjk9Had6XBl1gEfelY/J9BfEEHy9KX+Z/aNwFirGMa6I9hOtryeY/uTJBdo="
     - secure: "CfAXYEj34sBP3JuLguE9k6+0MXwVr924MrZ36dGMVp2Xvr2XvSyL4KEACHS9NalEl11wIOvrOmjfIj10KQI0G3SEsfDmSHPNYmkbtTQ0wrUVLFbis9PzKfCkmoDhUU0dZ0k3xMueDsHAkX6shKjJ5zxNhXwrarLgQE7TuyyBfLxnmekbkE5N8H1L38RLnF+2biyHql2SZggXwYCzRSLNQrqOWnG1XlT/N4wr/LvCxcpdeabs86AGAOUrMiISjm7oIWYrpAplPV1gtAkub0C5T1piCwaeJFcnHwXvg6pBFoSvf9Cw6mKkxrmPQ8QjH2RiZwWWJifIDSwYUCooQtYh5lqkr+HRWve6kgfbiSc99H/8i2LwV/jnJtFvrwC4jNmrt0haZH6KgJ3dTf3aOnpiCHV8gboSxZFMTJbdbc8XsNk0HYQgSysoRr0EVKs0iBqGSeqpFTHP6InroBsuYi9IDKMXGTIsJCatw0mtnMnacY2V8X8tmAoGj2bO+ueEI5hMFkdJRFx8YJoTVnv22dcHgYVghOJnF9gtOZxMaY7eUeNQFiU6ERC7LbC8TvkM+bK/VPQNyWsmyTxc2biyn22gOIXHGEyCO7I7u0v7lZew2dYVThxo73HDMyvoyxaCANHaPVV1bmgoGzX0tsJo4Y7eBo/tvkqS9a16MRoLvjDKVGk="
+    - DOCKER_USERNAME=joleesoftwire
```

Comment:
Very minor point, but you could use this variable in your docker tag/push commands as well.
And you could have an env var for `jol-todoapp`